### PR TITLE
Bug Fix: Fix pets menu detection for /ff

### PR DESCRIPTION
**Pull request by martimavocado**
for those questioning why the regex has a space, hypixel decided these should be the inv names
`Pets`
`Pets (1/2) `
`Pets (2/2) `

### CODE REVIEW

File: src/main/java/at/hannibal2/skyhanni/features/garden/fortuneguide/CaptureFarmingGear.kt [hunk 1]
```diff
@@ -49,6 +50,7 @@ class CaptureFarmingGear {
         "uniquevisitors.tierprogress",
         ".* §e(?<having>.*)§6/(?<total>.*)"
     )
+    private val petsRegex by RepoPattern.pattern("garden.fortuneguide.petsmenu", "Pets(?: \\(\\d+/\\d+\\) )?")
```
Comment (CalMWolfs) [inline src/main/java/at/hannibal2/skyhanni/features/garden/fortuneguide/CaptureFarmingGear.kt]:
```suggestion
    private val petMenuPattern by patternGroup.pattern(
        "petsmenu", 
        "Pets(?: \\(\\d+/\\d+\\) )?"
    )
```

File: src/main/java/at/hannibal2/skyhanni/features/garden/fortuneguide/CaptureFarmingGear.kt [hunk 2]
```diff
@@ -127,9 +129,9 @@ class CaptureFarmingGear {
         val farmingItems = farmingItems ?: return
         val outdatedItems = outdatedItems ?: return
         val items = event.inventoryItems
+        if (petsRegex.matches(event.inventoryName)) pets(farmingItems, items, outdatedItems)
```
Comment (CalMWolfs) [inline src/main/java/at/hannibal2/skyhanni/features/garden/fortuneguide/CaptureFarmingGear.kt]:
```suggestion        
        petMenuPattern.matchMatcher(event.inventoryName) {
            pets(farmingItems, items, outdatedItems)
            return
        }
```

**Review by CalMWolfs — CHANGES_REQUESTED** [review]
idk if what i did works as i made the suggestions on github but it should be formatted like this

File: src/main/java/at/hannibal2/skyhanni/features/garden/fortuneguide/CaptureFarmingGear.kt [hunk 3]
```diff
@@ -49,6 +49,7 @@ class CaptureFarmingGear {
         "uniquevisitors.tierprogress",
         ".* §e(?<having>.*)§6/(?<total>.*)"
     )
+    private val petMenuPattern by patternGroup.pattern("garden.fortuneguide.petsmenu", "Pets(?: \\(\\d+/\\d+\\) )?")
```
Comment (CalMWolfs) [inline src/main/java/at/hannibal2/skyhanni/features/garden/fortuneguide/CaptureFarmingGear.kt]:
```suggestion
    private val petMenuPattern by patternGroup.pattern(
        "petsmenu", 
        "Pets(?: \\(\\d+/\\d+\\) )?"
    )
```
You still need to change the pattern key.
Also please use this format so it is easier to read and will match the rest of the codebase once my other pr is merged

**Review by CalMWolfs — CHANGES_REQUESTED** [review]

**Review by CalMWolfs — APPROVED** [review]

**Review by hannibal002 — APPROVED** [review]

**Pull request merged**